### ISSUE TITLE
[Rogue] Cloak vs hunter's mark and serpent sting

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#5718]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you use cloak hunters can put mark you and serpent sting
**Expected behaviour**: Tell us what should happen instead
They can not do so when rogue cloaked, well 90% of the time.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

https://youtu.be/qrTS8e6j9lA
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Johanes022** [comment]
Hunters serpent sting is same phisycal ability that applies magical debuff same as for example plague strike that DKs have.

**Comment by Jiaku** [comment]
magical part should be resisted 90% of the time, physical should go through. Speaking of DKs, their aids go thorugh cloak as well, all of them. I would report it but knowing how fast devs fixing stuff, i don't wanna bother.
